script/create_selenium_db.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+
+set -o errexit
```

Comment:
It might be safer to enable all the error detection options then:
* `nounset` detects using unset variables (whether that is due to a coding mistake or a script misuse).
* `pipefail` detects failures of piped commands (even though `echo` is unlikely to fail here).
```suggestion
set -o errexit -o nounset -o pipefail
```